gravitee-am-ui/src/app/services/gio-license.service.ts

```diff
@@ -0,0 +1,164 @@
+      'The SAML 2.0 identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  'am-idp-ldap': {
+    utm: 'provider_ldap',
+    image: 'assets/gio-ee-unlock-dialog/am-idp-ldap.svg',
+    description:
+      'The LDAP identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  'am-idp-kerberos': {
+    utm: 'provider_kerberos',
+    image: 'assets/gio-ee-unlock-dialog/am-idp-kerberos.svg',
+    description:
+      'The Kerberos identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  'am-idp-azure-ad': {
+    utm: 'provider_azure_ad',
+    image: 'assets/gio-ee-unlock-dialog/am-idp-azure-ad.svg',
+    description:
+      'The Azure AD identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  'am-idp-france-connect': {
+    utm: 'provider_france_connect',
+    image: 'assets/gio-ee-unlock-dialog/am-idp-france-connect.svg',
+    description:
+      'The France Connect identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  'am-idp-cas': { // utm_source=oss_am&utm_medium=provider_cas&utm_campaign=oss_am_to_ee_am
+    utm: 'provider_cas',
+    image: 'assets/gio-ee-unlock-dialog/am-idp-cas.svg',
+    description:
+      'The CAS provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  // todo: Missing information
+  'am-idp-gateway-handler-saml': {
+    utm: 'provider_saml',
+    image: 'assets/gio-ee-unlock-dialog/gravitee-ee-upgrade.svg',
+    description:
+      'The SAML 2.0 identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  'am-idp-http-flow': {
+    utm: '',
+    image: 'assets/gio-ee-unlock-dialog/gravitee-ee-upgrade.svg',
+    description:
+      'The HTTP Flow identity provider is part of Gravitee Enterprise. Identity providers allow you to configure authentication methods familiar to your users and comply with your security requirement.',
+  },
+  // Resources
+  'am-smtp': { // todo: This resource is missing
+    utm: '',
+    image: 'assets/gio-ee-unlock-dialog/am-smtp.svg',
+    description:
+      'The SMTP resource is part of Gravitee Enterprise. Resources allow you to easily reuse some settings.',
+  },
```

Comment:
Still need to understand why this is here.
Since MFA email is not enterprise the resource should still be accessible